### ISSUE TITLE
Doctrine 1.1.6

### ISSUE THREAD

**Issue by flerow** [doctrine/doctrine1#80]
Hi there,
I need original source files of Doctrine 1.1.6, is there any possibility to get those?
I've searched through web, but unfortunatelly I found nothing. The problem is that, in our project repository we have modified version of Doctrine 1.1.6, and we need to upgrade. First we should check if there are any crucial changes. Saddly, changes where made before the source files where added to repository, so we don't have a clue, how it loooked like before. Thank you for help.

**Comment by alcaeus** [comment]
Looks like the issue got lost - we don't really expect issues on Doctrine 1. That said, the initial import in be13a7d68314db4409af0d4d7daa42d9615a8396 imported a changelog for 1.2.1, so the old tag may have gotten lost during the transition from SVN to GitHub. Maybe @jwage has some old stuff lying around, but I couldn't find anything.

**Comment by j13k** [comment]
I'm quite late coming to this, but a while back I did a separate `git svn` clone of Doctrine 1 from the original Subversion repositories before they were removed. This was to support a legacy project that was tracking Doctrine 1 from before the earliest branch that was available in this 'official' repo.

I didn't end up using it, but feel free to clone from here:

https://github.com/j13k/doctrine1

Of course, I can't make any guarantees as to the quality of the repository or the conversion.

The version 1.1.6 tag is here:

https://github.com/j13k/doctrine1/tree/1.1.6
